zsh/setup.zsh

```diff
@@ -0,0 +1,77 @@
+#! /bin/zsh
```

Comment:
difference between this and init script idea?

Reply:
init script is for downloading repo, this script is for after oh-my-zsh is installed?